packages/debian/rules

```diff
@@ -6,7 +6,7 @@ INIT_SYSTEM ?= systemd
 export PYBUILD_INSTALL_ARGS=--init-system=$(INIT_SYSTEM)
 
 %:
-	dh $@ --with python3 --buildsystem pybuild
+	dh $@ --buildsystem meson
```

Comment:
Why not use pybuild, which supports the meson-python backend?

I'm curious about the pros/cons between these two approaches: speed, compatibility, differences in packaged content, etc.